doc/migrations/4.2.0_5.0.0.md

```diff
@@ -0,0 +1,59 @@
+## DB migrations
+
+The migrations scripts for Postgres, MySQL, MSSQL can be found in the [`priv/migrations`](../../priv/migrations/) directory. Please remember to Provide the existing server domain for the `server` column instead of the localhost.
```

Comment:
```suggestion
The migrations scripts for Postgres, MySQL, MSSQL can be found in the [`priv/migrations`](../../priv/migrations/) directory. Please remember to provide the existing server domain for the `server` column instead of the localhost.
```